copy OE commit message here:
   commit 9456cdc1cf43e3ba9e6d88c63560c1b6fdee4359
   Author: Michael Krelin <hacker@klever.net>
   Date:   Tue May 29 12:27:45 2007 +0000

   apmd: prevent build from interferring with host kernel headers. Closes #1257

comment added by Kevin Tian <kevin.tian@intel.com>, 2010-07-13

Upstream-Status: Pending

Signed-off-by: Scott Garman <scott.a.garman@intel.com>

--- apmd-3.2.2.orig/Makefile
+++ apmd-3.2.2/Makefile
@@ -43,8 +43,7 @@
 
 CC=gcc
 CFLAGS=-O -g
-XTRACFLAGS=-Wall -pipe -I. -I/usr/src/linux/include -I/usr/X11R6/include \
-		-I/usr/src/linux-2.2/include -I /usr/src/linux-2.0/include \
+XTRACFLAGS=-Wall -pipe -I. \
 		-DVERSION=\"$(VERSION)\" \
 		-DDEFAULT_PROXY_NAME=\"$(PROXY_DIR)/apmd_proxy\"
 LDFLAGS=
